src/xy/xyEnsureFloat64.ts

```diff
@@ -0,0 +1,14 @@
+import { DataXY, NumberArray } from 'cheminfo-types';
+
+import { xEnsureFloat64 } from '../x/xEnsureFloat64';
+
+interface DataXYNumberArray {
+  x: NumberArray;
+  y: NumberArray;
+}
```

Comment:
How is that different from `DataXY<NumberArray>` ?

Reply:
I created https://github.com/cheminfo/cheminfo-types/pull/99 to allow NumberArray in the DataXY type